### PR TITLE
Add vpnkit.0.0.0

### PR DESCRIPTION
**Pull request by djs55**
This is a bit experimental.

### CODE REVIEW

File: packages/vpnkit/vpnkit.0.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  "asl"
+  "win-eventlog"
+  "fd-send-recv"
+  "logs"
+  "fmt"
+  "astring"
+  "mirage-flow" { >= "1.1.0" }
+  "mirage-types-lwt"
+  "ounit"
+  "alcotest"
```
Comment (samoht) [inline packages/vpnkit/vpnkit.0.0.0/opam]: this should probably be a `{test}` dependency (and same for `ounit`). Also not sure why `ounit` is still there, are still using it?

**Comment by avsm** [comment]
on osx:
```
# fatal: Not a git repository (or any of the parent directories): .git
# [ERROR] No package named hostnet found.
# W: No replace section found in file Makefile
# W: Cannot find source file matching module 'hostnet' in library hostnet.
# W: Use InterfacePatterns or ImplementationPatterns to define this file with feature "source_patterns".
# make: dylibbundler: No such file or directory
# make: *** [com.docker.slirp.tgz] Error 1
```

File: packages/vpnkit/vpnkit.0.0.0/opam (hunk 1 above)
Comment (djs55) [inline packages/vpnkit/vpnkit.0.0.0/opam]: Good catch -- I've removed `ounit` (unused) and marked `alcotest` as a test dependency.

File: packages/vpnkit/vpnkit.0.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  "named-pipe" { >= "0.4.0" }
+  "hvsock"
+  "asl"
+  "win-eventlog"
+  "fd-send-recv"
+  "logs"
+  "fmt"
+  "astring"
+  "mirage-flow" { >= "1.1.0" }
+  "mirage-types-lwt"
```
Comment (avsm) [inline packages/vpnkit/vpnkit.0.0.0/opam]: Could we upper bound mirage-types-lwt to <3.0? will work with mirage-dev as well then

**Pull request merged**